backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/externallyreferenced/InvoiceCandidateRepository.java

```diff
@@ -379,6 +379,8 @@ private I_C_Invoice_Detail createI_C_InvoiceDetail(final InvoiceCandidateId invo
 		invoiceDetailEntity.setLabel(invoiceDetailItem.getLabel());
 		invoiceDetailEntity.setNote(invoiceDetailItem.getNote());
 		invoiceDetailEntity.setPriceActual(invoiceDetailItem.getPrice());
+		invoiceDetailEntity.setPriceActual(invoiceDetailItem.getPrice());
+
```

Comment:
is this line a duplicate of the line above?

Reply:
it was, but now I've fixed it.